Feature/allow multiple linked orgs

cecemel opened the pull request:
See commit messages. We want to allow some subjects in multiple orgs.

cecemel committed:
Some subject types could belong to multiple orgs.

For some subject types, it might be ok to belong to
multiple orgs. For sure, e.g. adresses, in -debatable- cases, also
`person:Person`
Anyway, this commit, allows this flow, and based on the config, we can
say yes/no for multiple orgs.

However, there is a potential drawback in this implementation: if we need to take into
account both type AND path to determine whether yes/no it may be
shared accross organisations. Let's keep it simple here, and not
support this for now, and first see whether the business requires us
to do so.

cecemel committed:
added config

cecemel committed:
Bugfix: moving implies deleting in the source graph

cecemel committed:
debug statement

cecemel committed:
better debug statement

cecemel committed:
More robust handling of unknown types

benjay10 committed:
Linting fixes

benjay10 committed:
Use loglevels for logging

benjay10 commented:
I think there is an important case missing. I'll take some time testing this out and to make changes if needed.

benjay10 committed:
Fix case: 1 org and not allowed in multiple

Rewrote the if statement to include the data that belongs to one
organisation and that is not allowed in more than one organisation.
Also, if the config is missing, it now counts as false.

benjay10 reviewed (approved):
Had to make some adjustments and rewrote the `if` statement to include the case where there is only one fitting organisation and where the data is not allowed in multiple organisations.
